Xamarin.Forms.Platform.MacOS/Renderers/MediaElementRenderer.cs

```diff
@@ -0,0 +1,418 @@
+﻿using System;
+using AppKit;
+using AVFoundation;
+using AVKit;
+using CoreMedia;
+using Foundation;
+
+namespace Xamarin.Forms.Platform.MacOS
+{
+	public class MediaElementRenderer : ViewRenderer<MediaElement, NSView>
```

Comment:
In addition to use `NSView` instead of `UIView`, the Uri resolution and `AVPlayerView` instead of `AVPlayerViewController`, are there other changes between the macOS and iOS implementation?. I wonder if we could use the same class in both platforms just using a couple of compilation directives.

Reply:
I have also added AVPlayerLayer to handle Aspect changes.
Modified KeepScreenOn (totally replaced)
Modified Dispose method (added disposing layer and view).